src/leap/mail/imap/mailbox.py

```diff
@@ -514,13 +514,14 @@ def _get_imap_msg(messages):
                 d_imapmsg = []
                 for msg in messages:
                     d_imapmsg.append(getimapmsg(msg))
-                return defer.gatherResults(d_imapmsg)
+                return defer.gatherResults(d_imapmsg, consumeErrors=True)
 
             def _zip_msgid(imap_messages):
                 zipped = zip(
                     list(msg_range), imap_messages)
                 return (item for item in zipped)
 
+            # XXX not called??
```

Comment:
that question is isomorphic with "will we ever empty our (chili) backlog?". with the advantage that it's easier to cleanup when touching relating code because stuff doesn't get lost on a huge black hole. I personally have a vim plugin that allows me to navigate the pending tasks when I'm fixing or refactoring a module.